corehq/apps/reports/tasks.py

```diff
@@ -121,11 +123,15 @@ def get_report_queue(report):
 @task(ignore_result=True)
 def send_report(notification_id):
     notification = ReportNotification.get(notification_id)
+    owner = WebUser.get(notification.owner_id)
```

Comment:
Is this going to result in a bad performance, thought about something like `get_cached_property`, but it doesn't make much sense here. Any other ideas?